### DISCUSSION TITLE
Installing in Auto1111

### DISCUSSION THREAD

**Discussion by JungeWerther** [v8hid/infinite-zoom-automatic1111-webui#85] [Q&A] [answered]
Hey, I'm trying to install in Auto1111 on Colab.

when I install and restart the UI it looks like the extension is installed - but how to use it? I can't seem to find where.

Thanks :)

**Comment by v8hid** [comment, marked as answer]
@JungeWerther I didn't test on Google Colab before, you should see a new tab named Infinite zoom.
If it is not shown  for you check Google Colab output results for error.

**Comment by linchio** [comment]
same problem I don't understand what u mean with; "check Google Colab output results for error." I'm in local on my pc don't know where to look for: I don't have a google collab...I think?
sorry for the silly request.
